### PR TITLE
Some fixes for sol_network_get_hostname_address_info()

### PR DESCRIPTION
**Pull request by cabelitos**
@barbieri Could you please take a look?

### CODE REVIEW

File: src/lib/comms/include/sol-network.h [hunk 1]
```diff
@@ -319,8 +319,7 @@ bool sol_network_link_up(uint16_t link_index);
 struct sol_network_hostname_handle *
 sol_network_get_hostname_address_info(const struct sol_str_slice hostname,
     enum sol_network_family family, void (*host_info_cb)(void *data,
-    const struct sol_str_slice hostname,
-    struct sol_network_link_addr *sol_addr_list, uint16_t list_size),
+    const struct sol_str_slice hostname, const struct sol_vector *addrs_list),
```
Comment (barbieri) [inline src/lib/comms/include/sol-network.h]: you also need to update the documentation.

**Comment by barbieri** [comment]
LGTM, but fix the doc before :-)

**Comment by cabelitos** [comment]
Thank you for the review. I will update the doc and merge it. :)

**Comment by cabelitos** [comment]
merged

**Pull request closed without merging**